### ISSUE TITLE
make ExpressRouter replaceable

### ISSUE THREAD

**Issue by gasolin** [gasolin/webbybot#42]
to 4.x

**Comment by gasolin** [comment]
consider make it replaceable such as https://github.com/github/hubot/pull/1134/files

**Comment by ccoenen** [comment]
would this mean that express itself wouldn't have to be a dependency? Over half of this project's dependencies come in via express. The full left and bottom part of this graph are express. Only the top right stuff is the rest of webbybot. 

![grafik](https://cloud.githubusercontent.com/assets/124909/20881280/e8297b44-badb-11e6-99da-eb4c1a8d4c23.png)

http://npm.anvaka.com/#/view/2d/webbybot

**Comment by gasolin** [comment]
no, it means we'd usenewer express version to avoid related security issue.Now webbybot already pick different express version(4) compare to hubot(3)

**Comment by gasolin** [comment]
Actually hubot only use a small portion of express, so theoretically we can totally replace it. But it should be carefully done when compatibility is in concern.
